### PR TITLE
Enable alt dev setup

### PR DESCRIPTION
**Pull request by cuahsimarko**
### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

- [x] Automated Testing

- [x] Sufficient User and Developer Documentation

- [x] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
The only change in this PR is to add files that are not used by existing configurations.

### CODE REVIEW

**Comment by cuahsimarko** [comment]
This will enable me to contribute, without the complex hsctl commands, adaptation for macOS, or VirtualBox. All files are in my local setup, ignored by the .gitignore, until a wider discussion occurs on what my changes offer across the entire stack and dev process as well as release/deploy.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# Temporary for alternate dev setup
+conf_postgres
+conf_solr
+Dockerfile-defaultworker
+Dockerfile-hydroshare
+Dockerfile-postgis
+Dockerfile-solr
+
+# HydroShare Custom
```
Comment (cuahsimarko) [inline .gitignore]: Note that the only Dockerfile in use is for hydroshare and is called Dockerfile. The dockerfile-compose.yml is already ignored by the current setup, so conveniently covers my use case.

**Comment by cuahsimarko** [comment]
@hyi I fixed this in 23eac79a9 and also note that I found that the gitignore was wiping out my file so I fixed that, too.

**Comment by cuahsimarko** [comment]
I am not sure why Jenkins PR hasn't succeeded yet

**Comment by sblack-usu** [comment]
Because it takes 3 hours to run and someone pushed another branch several times before you

**Pull request merged**